src/paradigma/heart_rate/heart_rate_estimation.py

```diff
@@ -0,0 +1,185 @@
+import numpy as np
+from typing import Tuple
+
+from paradigma.config import HeartRateExtractionConfig
+from paradigma.heart_rate.tfd import nonsep_gdtfd
+
+def assign_sqa_label(ppg_prob, config: HeartRateExtractionConfig, acc_label=None) -> np.ndarray:
```

Comment:
```suggestion
def assign_sqa_label(ppg_prob: np.ndarray, config: HeartRateExtractionConfig, acc_label: np.ndarray=None) -> np.ndarray:
```

Reply:
I have ajdusted the ppg_prob suggestion but not for the acc_label since I got a type error from this assignment